src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -323,14 +373,19 @@ private void DisplayTemplateList(bool showAll = false)
 
                 foreach (ITemplateInfo info in grouping)
                 {
-                    if (info.Tags != null && info.Tags.TryGetValue("language", out string lang))
+                    if (info.Tags != null && info.Tags.TryGetValue("language", out ICacheTag languageTag))
                     {
-                        if (string.IsNullOrEmpty(Language) && string.Equals(defaultLanguage, lang, StringComparison.OrdinalIgnoreCase))
+                        foreach (string lang in languageTag.ChoicesAndDescriptions.Keys)
```

Comment:
I assume that `ChoicesAndDescriptions` can never be null? Haven't made it to where it's populated yet